templates/partials/answers/checkbox.html

```diff
@@ -1,20 +1,22 @@
 {% from "components/checkboxes/_macro.njk" import onsCheckboxes %}
 
-{% if answer.label %}
-  {% set checkbox_label = answer.label %}
-{% elif answer.options|length == 1 %}
-  {% set checkbox_label = '' %}
+{% if answer.label == None or answer.options|length == 1 %}
+  {{ onsCheckboxes({
+    "legend": question_title,
+    "id": answer.id,
+    "legendClasses": "u-vh",
+    "checkboxes": map_checkbox_config(form, answer),
+    "mutuallyExclusive": mutuallyExclusive,
+    "error": error
+  }) }}
```

Comment:
There is now duplication of the `onsCheckboxes` call, so they could un-knowingly diverge. I think this should follow the same pattern as `radio.html` where the macro config that is the same is set, and then an attribute (`checkboxesLabel` in this case) would be set if applicable before calling the macro.

Reply:
Added new functional tests with new test schema with all those features you requested plus checkbox template is now using syntax as in radio template. Thanks!